ui/src/components/imquery/EditProperty.vue

```diff
@@ -1,39 +1,65 @@
 <template>
   <div class="property-container">
     <InputText v-if="selectedProperty" v-model="selectedProperty.propertyName" class="w-full md:w-14rem" disabled />
-    <ConceptSelect
+    <!-- <ConceptSelect
       v-if="selectedProperty?.propertyType === 'class' || selectedProperty?.propertyType === 'node'"
       :datatype="selectedProperty.valueType"
       :property="property"
       :data-model-iri="dataModelIri"
       class="concept-select"
-    />
+    /> -->
+    <span v-if="selectedProperty?.propertyType === 'class' || selectedProperty?.propertyType === 'node'">
+      <InputText value="is" disabled />
```

Comment:
This feels like the wrong component is being used. Disabled inputtext has limitations. User can't highlight/copy text etc on disabled. Create a textDisplay component or use the generics in shared components.